lit_tests/kernel/wave/scheduling.py

```diff
@@ -120,45 +120,46 @@ def test_gemm_pipelined():
         # CHECK-NEXT: %rotating_reg_1
         # CHECK-NEXT: %rotating_reg_2
         # CHECK-NEXT: %rotating_reg_3
-        # CHECK-NEXT: %read_shared_M_0_N_0_K_0
-        # CHECK-NEXT: %read_shared_M_0_N_0_K_1
+        # CHECK-NEXT: %read_4_shared_M_0_N_0_K_0
+        # TODO: Why did this change from K_1 to K_0???
+        # CHECK-NEXT: %read_2_shared_M_0_N_0_K_0
         # CHECK-NEXT: %read_21
         # CHECK-NEXT: %read_22
         # CHECK-NEXT: %scheduling_group_barrier
         # CHECK-SAME: ({Operation.READ_SHARED: 2, Operation.READ_GLOBAL: 2}, 0)
-        # CHECK-NEXT: %read_shared_M_0_N_1_K_0
-        # CHECK-NEXT: %read_shared_M_0_N_1_K_1
+        # CHECK-NEXT: %read_4_shared_M_0_N_1_K_0
```

Comment:
Also these are f"{dim_name}_{dim_val}" instead of f"{dim_name}:{dim_val}". It kind of looks like these are getting process by expansion, which was stripping the numeric disambiguator and adding "_shared" and then later something else was doing its own expansion and then adding a numeric disambiguator, but actually treating the last dim value as the thing to increment.